### ISSUE TITLE
Reload Configuration

### ISSUE THREAD

**Issue by scowcron** [amici-ursi/ImagesOfNetwork#58]
I'd be great to be able to send a HUP to the bot and have it reload it's configuration without killing the bot.

**Comment by amici-ursi** [comment]
agreed. i was trying to think of a way to allow trusted people to admin the bot. messages via PM might be a relatively easy way.

the alternative being logged console access, which seems like a nightmare for me to set up.

**Comment by scowcron** [comment]
I've got a couple of ideas how to do that. either a very simple api or some rpc i think would be the way to go. i'd think we'd want to run it (let's call it the manager) as a separate process. that's really a whole different task though.
